feat:added profile page

isha2510 opened the pull request:
feat:added profile page
feat:added loader

isha2510 committed:
feat:added profile page
-added loader

sagar-kale commented on client/src/app/components/Profile/Profile.tsx:
```diff
@@ -1,17 +1,72 @@
   }
 
   return (
     isAuthenticated && (
```
no need to check is Authenticated. it will not inside this if its in private route

sagar-kale commented on client/src/app/components/Profile/Profile.tsx:
```diff
@@ -1,17 +1,72 @@
+import Loader from "../Loader/Loader";
 
 const Profile = () => {
   const { user, isAuthenticated, isLoading } = useAuth0();
```
user user! which will denote user always have value in case of logged in

isha2510 committed:
feat: added the logo
- changed background color for navbar
- changed button css on hover
- changed the vite logo
fix: remove unecessary auth check

isha2510 committed:
fix: thrown error if user is null

sagar-kale reviewed (approved):
ok
